src/app/team/member.component.html

```diff
@@ -0,0 +1,21 @@
+<mat-card>
+  <img mat-card-image [src]="member.avatar" />
+  <mat-card-header>
+    <mat-card-title-group>
+      <mat-card-title>{{member.name}}</mat-card-title>
+      <mat-card-subtitle>{{member.role}}</mat-card-subtitle>
+    </mat-card-title-group>
+  </mat-card-header>
+  <mat-card-content>
+    <p>
+      {{member.bio}}
+    </p>
+  </mat-card-content>
+  <mat-card-actions>
+    <app-social-sharing
+      [githubUrl]="member.githubUrl"
+      [twitterUrl]="member.twitterUrl"
+      [linkedinUrl]="member.linkedinUrl">
```

Comment:
`email` is missing here but its part of the member interface.

Reply:
Fixed - removed that unused property from the model